data/aliases.ts

```diff
@@ -2355,188 +2522,335 @@ export const Aliases: import('../sim/dex').AliasesTable = {
 	krilo: "Krilowatt",
 	libra: "Equilibra",
 	mala: "Malaconda",
-	maw: "Miasmaw",
 	megacruci: "Crucibelle-Mega",
 	navi: "Naviathan",
 	nect: "Necturna",
 	ohmagod: "Plasmanta",
 	plas: "Plasmanta",
 	raja: "Saharaja",
-	rev: "Revenankh",
+	reve: "Revenankh",
 	roak: "Pyroak",
 	smoko: "Smokomodo",
 	snael: "Snaelstrom",
+	soil: "Colossoil",
 	strata: "Stratagem",
+	tomo: "Tomohawk",
 	train: "Chuggalong",
 	venomicone: "Venomicon-Epilogue",
 	venomiconp: "Venomicon",
 	venomiconprologue: "Venomicon",
 	volk: "Volkraken",
 };
 
-/** This is what lets you type "tb"/"tbold"/"bolt" to find "thunderbolt" in the
+/** This is what lets you type "tb"/"tbolt"/"bolt" to find "thunderbolt" in the
   * teambuilder (and PSdex) search. */
 export const CompoundWordNames = [
 	// species
 	'Aegi-slash',
```

Comment:
```suggestion
	'Aboma-snow',
	'Aegi-slash',
```

(You shouldn't have removed it)